ports/openssh/vcpkg.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "openssh",
+  "version": "9.1",
+  "port-version": 1,
+  "description": "OpenSSH is the premier connectivity tool for remote login with the SSH protocol.",
+  "homepage": "https://www.openssh.com",
+  "license": "BSD-3-Clause",
```

```suggestion
  "license": "BSD-3-Clause",
  "supports": "linux",
```

>This application build on Linux

If only linux is supported, please add `supports`.